cmd/operator/main.go

```diff
@@ -32,13 +33,17 @@ func main() {
 		namespace       string
 		metricsAddr     string
 		healthProbeAddr string
+		images          k8sflag.MapStringString
+		versions        k8sflag.MapStringString
 
 		setupLog = ctrl.Log.WithName("setup")
 	)
 
 	flag.StringVar(&namespace, "namespace", "default", "The namespace in which the operator runs")
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&healthProbeAddr, "health-probe-bind-address", ":8081", "The address the health probe endpoint binds to.")
+	flag.Var(&images, "images", "Images to use for containers managed by the observability-operator.")
```

Comment:
?
```suggestion
	flag.Var(&images, "images", "Images to use for containers managed by the cluster-observability-operator.")
```

Reply:
hmm good point, but I'd say maybe we should go with `managed by the  operator`? This way we don't get weird doc strings that add confusion around upstream and downstream.
And it should be pretty clear which operator is meant.

Reply:
:+1: